_includes/google-analytics.html

```diff
@@ -1,10 +1,9 @@
 <!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id={{ site.google_analytics }}"></script>
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-170203794-1"></script>
```

Comment:
i wonder why the variable did not work. would be preferable to have it as a variable.

Reply:
I changed it to variable in update #2